Stripe/STPSource.m

```diff
@@ -149,7 +149,60 @@ - (BOOL)isEqualToSource:(STPSource *)source {
     return [self.stripeID isEqualToString:source.stripeID];
 }
 
-#pragma mark STPAPIResponseDecodable
+#pragma mark - Description
+
+- (NSString *)description {
+    NSString *statusDescription;
+
+    switch (self.status) {
```

Comment:
same comment as above, can we pull this out into `stringFromStatus` and couple it with `statusFromString`?